plugins/inputs/ssl_cert/ssl_cert.go

```diff
@@ -0,0 +1,155 @@
+package ssl_cert
```

Comment:
I will keep the underscore in the package name because the name appears in the config file and I believe that it improves readability of the config file.